include/RMGPhysics.hh

```diff
@@ -100,11 +102,9 @@ class RMGPhysics : public G4VModularPhysicsList {
     StepCutStore fStepCuts;
     StepCutStore fStepCutsSensitive;
     bool fConstructOptical = false;
-    bool fUseLowEnergyEM = false;
-    bool fUseHadPhys = false;
     bool fUseThermalScattering = false;
-    LowEnergyEMOption fLowEnergyEMOption = LowEnergyEMOption::kLivermore;
-    HadronicPhysicsListOption fHadronicPhysicsListOption = HadronicPhysicsListOption::kShielding;
+    LowEnergyEMOption fLowEnergyEMOption = LowEnergyEMOption::kNone;
```

Comment:
```suggestion
    LowEnergyEMOption fLowEnergyEMOption = LowEnergyEMOption::kLivermore;
```

before #87, Livermore was always constructed (with no way to turn it off). This changed in the mentioned PR, but I am not sure what is intended/expected. _Just noting this down for later discussion..._